courses/models.py

```diff
@@ -55,35 +56,37 @@ def with_text_id(self, text_id):
         """Applies a filter for the Program's readable_id"""
         return self.filter(readable_id=text_id)
 
-
-class ProgramManager(models.Manager):  # pylint: disable=missing-docstring
-    def get_queryset(self):
-        """Manager queryset"""
-        return ProgramQuerySet(self.model, using=self._db)
-
-    def live(self):
-        """Returns a queryset of Programs with live=True"""
-        return self.get_queryset().live()
-
-    def with_text_id(self, text_id):
-        """Returns a queryset filtered by text id (i.e.: readable_id)"""
-        return self.get_queryset().with_text_id(text_id)
+    def for_product_serializer(self):
```

Comment:
Is this used anywhere?

Reply:
I think this was a holdover from one of my various attempts at structuring the querysets/serializers.